Gateway/Command/CreatePayment.php

```diff
@@ -48,6 +48,7 @@ public function execute(array $commandSubject)
             'orderId' => $orderId,
             'method' => $method,
             'type' => $type,
+            'captureType' => 'AUTOMATIC_PLUGIN',
```

Comment:
Locally, line 57 is overwriting with AUTOMATIC_CHECKOUT, so it's not working as intended. I think we should only overwrite capture type if its not AUTOMATIC_PLUGIN/CHECKOUT